lib/utils.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import { quote } from 'shell-quote';
+
+/**
+ * Add quotes to shell command arguments if the particular
+ * argument contains one or more whitespace characters
+ * and is not already quoted.
+ *
+ * @param {Array<*>|*} args [[]] - The array of arguments to be quoted
+ *                                 or a single argument value
+ *
+ * @returns {Array<*>} The args array with quoted items.
+ * Other items are simply left untouched.
+ */
+function quoteSpawnArguments (args = []) {
+  if (_.isEmpty(args)) {
+    return args;
+  }
+  if (!_.isArray(args)) {
+    args = [args];
```

Comment:
This seems wrong. Passing in a single, non-array, argument will result in an array being returned.